CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+- [#5] Allow for call-level age overrides as specified in the README.
+- [#4] Bump various dev-dependencies
```

Comment:
Doesn't this PR make #4 useless?

Reply:
sure, unless we want them to be separate and more targeted in scope.

Reply:
Did the merge separately.